app/registration/views/patient-summary.html

```diff
@@ -3,7 +3,7 @@
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.program">{{'COMMON_PROGRAM_ENROLLMENT_TITLE' | translate}}</a></li>
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.visits">{{'COMMON_VISIT_HISTORY_TITLE' | translate}}</a></li>
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.services">{{'COMMON_CLINIC_SERVICES_TITLE' | translate}}</a></li>
-        <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.alerts">{{'COMMON_ALERT_TITLE' | translate}}</a></li>
+        <!--<li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.alerts">{{'COMMON_ALERT_TITLE' | translate}}</a></li>-->
```

Comment:
Not enough @ningosi, we'll probably need to tell the system not only to hide the deactivated app but also to not load it, for performance matters. But for now it will be enough to just hide it in the view, something like: `ng-show="app.active"`, since the apps are not developed yet.